packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -237,10 +242,17 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     async () => this.epochTracker.fetchAndParseAsJSON<IBlob>(url, { headers }),
                 );
             });
-            blob = response.content;
+            // Verify that we haven't populated the cache in the meantime from a simultaneous read
+            blob = this.blobCache.get(blobid);
```

Comment:
Unless you plan to move this logic to factory scope (i.e. increase scope), I do not think caching these blobs makes sense.
We never read them again

Reply:
Sounds good, will remove in next iteration.